pkgs/applications/networking/instant-messengers/teamspeak/server.nix

```diff
@@ -59,43 +59,9 @@ in stdenv.mkDerivation rec {
     description = "TeamSpeak voice communication server";
     homepage = "https://teamspeak.com/";
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
+    # See distribution-permit.txt for a confirmation that nixpkgs is allowed to distribute TeamSpeak.
     license = licenses.unfreeRedistributable;
     platforms = platforms.linux;
     maintainers = with maintainers; [ arobyn gerschtli ];
   };
 }
-
-/*
-License issues:
```

Comment:
This was the same as in client.nix, so I dropped it and added the note above. But the license in `meta` is different, so for me it looks like the note is not necessary.